setup.py

```diff
@@ -520,7 +522,8 @@ def get_vllm_version() -> str:
     elif _is_tpu():
         version += f"{sep}tpu"
     elif _is_cpu():
-        version += f"{sep}cpu"
+        if envs.VLLM_TARGET_DEVICE == "cpu":
+            version += f"{sep}cpu"
```

Comment:
Why do we need this guard now for cpu?

Reply:
This is a little tricky here. Like L491 does,  it's used for check if the `VLLM_TARGET_DEVICE` is set by hand or not. 
`_is_cpu()` means `VLLM_TARGET_DEVICE` is set to `cpu` by hand or by code logic.  `envs.VLLM_TARGET_DEVICE == "cpu"` means it's set by hand.

When users build vllm by hand like `VLLM_TARGET_DEVICE=cpu python setup.py install` the version will will contains the suffix `cpu` which is the origin behavior.

When use `pip install vllm`, the version will not contain the suffix  `cpu`. If it contains the suffix `cpu` , the error `has inconsistent version: expected '0.7.2', but metadata has '0.7.2+cpu'` will be raised by pip like the commit message say.